internal/fs/inode/dir.go

```diff
@@ -126,6 +126,10 @@ type DirInode interface {
 		ctx context.Context,
 		name string,
 		isImplicitDir bool) (err error)
+
+	// localFileEntries lists the local files present in the directory.
+	// Local means that the file is not yet present on GCS.
+	LocalFileEntries(localFileInodes map[Name]Inode) (localEntries []fuseutil.Dirent)
```

Comment:
should we call it getLocalFileEntries or ListLocalFileEntries. what is the go recommendation?

Reply:
The recommendation is to avoid "Get" prefix and to prefer starting the name with the noun directly go/go-style/decisions#getters